scripts/Excel_diff.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
Should this be in `raven/tests/scripts/TestHarness/testers`?

Reply:
@PaulTalbot-INL since it's a general script for use outside of the testing harness, I instructed @wenchichenginl to put it in `raven/scripts/` but let me know if you think it should only live within the testing harness.